ext.php

```diff
@@ -20,8 +20,8 @@
 */
 class ext extends \phpbb\extension\base
 {
-	/** @var string Require phpBB 3.1.3 due to the use of new exception classes */
-	const PHPBB_MIN_VERSION = '3.1.3';
+	/** @var string Require phpBB 3.1.5 due to the use of new template events */
+	const PHPBB_MIN_VERSION = '3.1.4';
```

Comment:
A separate PR will change this back for 3.1.5 when released (whenever that happens).